pkg/v1/tkg/web/src/app/shared/service/azure-wizard-form.service.ts

```diff
@@ -51,6 +51,12 @@ export class AzureWizardFormService extends WizardFormBase {
         if (!method) {
             return throwError({ message: `Unknown data source ${source}` });
         }
+        if (!this.region) {
+            const msg = 'WARNING: azure-wizard-form.retrieveDataForSource() received event ' +
```

Comment:
I like having some console logging because it helps debug a production build. We have some inconsistency in how we form the messages, for example, ===> WARNING and just WARNING.

Maybe we should consider a lightweight logging service that could help with consistency in outputting console logs and perhaps writing other detailed logs to a file. This would of course be done in another PR I imagine. What do you think?